db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```

Comment:
- [ ] There is an error raised while trying to get to the API endpoint you created, That happened because you forgot to add 5 greetings seeds here, so please fix that.

![image](https://user-images.githubusercontent.com/68796232/223901786-abc05280-a2b0-4b6a-b9e8-08e71f82c274.png)

When I added these seeds everything work as expected:
```ruby
greeting1 = Message.create!(greeting: "Hello")
greeting2 = Message.create!(greeting: "Aloha")
greeting3 = Message.create!(greeting: "Hola")
greeting4 = Message.create!(greeting: "Bonjour")
greeting5 = Message.create!(greeting: "Ciao")
``` 

![image](https://user-images.githubusercontent.com/68796232/223902092-a2d3591f-dd45-4cab-9f57-87a4844a8e92.png)

Reply:
Thank you for taking out time to review the project @Omar-Muhamad. I had populated the database in my local environment and forgot about seeding it. Thank you for pointing that out.